frontend/src/pages/RoadTripPage/RoadTripPage.jsx

```diff
@@ -1,26 +1,20 @@
 import React from "react";
 import SideBar from "../../components/SideBar/SideBar";
+import TopBar from "../../components/TopBar/TopBar";
```

Comment:
maybe merge with main before you merge this PR because main has `import TopBar from "../../components/TopBar/TopBar";` added as line 2, and I think this PR will result in duplicate imports on this page.